scripts/run_benchmarks.py

```diff
@@ -40,25 +38,15 @@ def compute_ppl(
     model: AutoModelForCausalLM,
     tokenizer: AutoTokenizer,
     stride: int = 512,
```

Comment:
Do we need to adjust stride as well?

Reply:
I think 512 is still reasonable. We'd only need to adjust it if the benchmark is too slow